plugins/node/opentelemetry-instrumentation-pg/src/pg.ts

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  isWrapped,
+  InstrumentationBase,
+  InstrumentationConfig,
+  InstrumentationNodeModuleDefinition,
+} from '@opentelemetry/instrumentation';
+
+import {
+  context,
+  getSpan,
+  Span,
+  SpanKind,
+  StatusCode,
+} from '@opentelemetry/api';
+import * as pgTypes from 'pg';
+import * as pgPoolTypes from 'pg-pool';
+import {
+  PgClientExtended,
+  NormalizedQueryConfig,
+  PostgresCallback,
+  PgPoolExtended,
+  PgPoolCallback,
+} from './types';
+import * as utils from './utils';
+import { AttributeNames } from './enums';
+import { VERSION } from './version';
+
+export interface PostgresInstrumentationConfig {
+  ignoreMethods?: string[];
```

Comment:
not sure why we have http related options here

Reply:
just blindness leftover,